views/lexicon/includes/notifications.html.twig

```diff
@@ -11,7 +10,7 @@
         <div class="notifications-container dropdown__menu"><!--
             --><div class="notifications__header">
                 <h1 class="notifications-title">Notifications</h1>
-              <a href="#" class="notifications-dismiss">dismiss all</a>
+                <button class="notifications-dismiss btn--naked">dismiss all</button>
```

Comment:
Assume styling is ok here - thought btn--naked would have extra padding etc vs a link.

Reply:
Yeah styling is fine here

Reply:
![screen shot 2018-07-02 at 10 13 12](https://user-images.githubusercontent.com/18653/42155541-9167990c-7de0-11e8-9422-c925ace8e00d.png)